tex/gsp-default.tex

```diff
@@ -210,8 +210,12 @@
 \grecreatedim{initialraise}{0 cm}{scalable}%
 % Space between lines in the annotation
 \grecreatedim{annotationseparation}{0.05cm}{scalable}%
-% Amount to raise (positive) or lower (negative) the annotations from the default position (base line of top annotation aligned with top line of staff)
+% Amount to raise (positive) or lower (negative) the annotations from the default position
 \grecreatedim{annotationraise}{0cm}{scalable}%
+% Space between lines in the commentary
+\grecreatedim{commentaryseparation}{0.05cm}{scalable}%
+% Amount to raise (positive) or lower (negative) the commentary from the default position (base line of bottom commentary aligned with top line of staff)
+\grecreatedim{commentaryraise}{0.4cm}{scalable}%
```

Comment:
1. I picked 0.4cm to be reasonably close to the existing behavior.  I can easily shrink it if something tighter is more desirable.
2. I actually don't like this idea because it provides extra pathways for the unexpected to happen.  I'd much prefer the user having a group bound `\grechangedim` command for changing `commentaryraise` for just one score.